libs/contact/src/actions/contact.actions.ts

```diff
@@ -0,0 +1,44 @@
+import { Action } from '@ngrx/store';
+
+export enum DaffContactActionTypes {
+  ContactSubmitAction = '[Daff-Contact] Contact Submit Action',
+  ContactCancelAction = '[Daff-Contact] Contact Cancel Action',
+  ContactSuccessSubmitAction = '[Daff-Contact] Contact Success Submit Action',
+  ContactFailedSubmitAction = '[Daff-Contact] Contact Failed Submit Action',
+  ContactRetryAction = '[Daff-Contact] Contact Retry Action',
+  ContactResetAction = '[Daff-Contact] Contact Reset Action'
+}
+
+export class DaffContactSubmit<T> implements Action {
+  readonly type = DaffContactActionTypes.ContactSubmitAction
+
+  constructor(public payload: T) { }
+}
+
+export class DaffContactRetry<T> implements Action {
+  readonly type = DaffContactActionTypes.ContactRetryAction
+
+  constructor(public payload: T) { }
+}
+export class DaffContactFailedSubmit implements Action {
+  readonly type = DaffContactActionTypes.ContactFailedSubmitAction
+
+  constructor(public payload: string[]) { }
+}
+export class DaffContactCancel implements Action {
+  readonly type = DaffContactActionTypes.ContactCancelAction
+}
+export class DaffContactSuccess implements Action {
```

Comment:
Yes you're totally right, this file could be removed in this PR. I initially created it with the creation of the library so that the testing suite would run and not error.